### DISCUSSION TITLE
How to interact with the summary?

### DISCUSSION THREAD

**Discussion by LorenzoBettini** [nvim-neotest/neotest#464] [Q&A]
Maybe I'm missing something but once the summary is open, if it's all collapsed, I see no way to "expand all": I have to manually expand each single node.
Moreover, is there a way to jump from one of the selected test in the summary to the corresponding test source file?

**Comment by fredrikaverpil** [comment]
You can hit `?` for a legend of keys you can press there.

**Reply by dennypenta** [reply]
It took me some time to find it in the source code, I would put it in the readme
